src/components/Web3Status/index.tsx

```diff
@@ -43,6 +45,10 @@ function Web3StatusInner() {
 
     const toggleWalletModal = useWalletModalToggle()
 
+    const onAccountClick = () => {
+        sendData({ event: 'goto_page', action: 'goto_address' })
```

Comment:
shouldn't address value be logged with the event ?

Reply:
@drptiago ?

Reply:
Yes, please. 

Send the address, so we will know which button was clicked @johnsmith-gooddollar @L03TJ3 

goto_dashboard
goto_swap
goto_stake
goto_portfolio
goto_claim
goto_micro_bridge
goto_wallet
goto_bridge
goto_docs

Reply:
@L03TJ3 how about this ? don't see any changes related in the last commit